.github/workflows/docs.yml

```
@@ -24,7 +24,7 @@ jobs:
           -D STEPS=clean\;configure\;docs
           -S FairCMakeModules_test.cmake
       - name: Deploy
-        uses: JamesIves/github-pages-deploy-action@v4.6.0
+        uses: JamesIves/github-pages-deploy-action@v4.6.1
```

@dennisklein: I think, we should just use the latest patch version (in the hope that they get the semantic versioning correct). What do you think?

```suggestion
        uses: JamesIves/github-pages-deploy-action@v4.6
```

---

Why? Dont we want to benefit from bug fixes?

---

I had assumed that `4.6` means "take the latest patch release in the `4.6.*` range? So it should get us all bug fixes in 4.6 without even getting a PR like this one?

Getting a PR like this one for feature updates or even breaking changes is good!

---

Ah yes, now I understand. I agree, let's use the least specific version that makes sense.